portals/application/configs/klear/model/TransformationRulesetGroupsTrunks.yaml

```diff
@@ -84,6 +84,7 @@ production:
               - calling_code
             template: '%name% (+%calling_code%)'
           order: name_${lang}
+      defaultValue: 70
```

Comment:
I don't like hardcoding values. For DDI's we use default company's countryId taken from auth (check DDIList.yaml), but Brands have no countryId, so I won't argue for this.